### DISCUSSION TITLE
[FEATURE REQ] Notifications per filter or Indexer

### DISCUSSION THREAD

**Discussion by ghost** [autobrr/autobrr#1653] [Ideas]
Hello,
would it be possible to add different telegram bots and have each bot take the messages from an indexer?

Regards

**Comment by zze0s** [comment]
We have discussed before about connecting notifications to filters and that's in some todo somewhere. Would that fit your needs? You could do single filter per indexer and that would work.

**Comment by ghost** [comment]
I would like to use a bot for each tracker so that I can mute a specific tracker's push notifications and still receive them.
Currently, you can add multiple bots, but they all send the whole filters. It would be great if you could choose for each bot which filters or trackers should be sent to which bot.

**Comment by onedr0p** [comment]
This would be nice if you have a filter that you wouldn't want to get notified on for actions and others you do. 👍🏼

**Comment by ghost** [comment]
I would also like to be able to control the priority of the notification on a per filter basis (for pushover).

**Comment by zze0s** [comment]
Work was started in this PR https://github.com/autobrr/autobrr/pull/2124

**Comment by luckylittle** [comment]
This is now implemented and can be closed
